src/openforms/emails/digest.py

```diff
@@ -113,10 +118,10 @@ class InvalidRegistrationBackend:
 
     @property
     def admin_link(self) -> str:
-        form_admin_url = reverse(
-            "admin:forms_form_change", kwargs={"object_id": self.form_id}
+        form_relative_admin_url = furl(
+            reverse("admin:forms_form_change", kwargs={"object_id": self.form_id})
         )
-        return form_admin_url
+        return build_absolute_uri(form_relative_admin_url.url)
```

Comment:
```suggestion
        return build_absolute_uri(reverse("admin:forms_form_change", kwargs={"object_id": self.form_id}))
```
No need for `furl` here